sources/osgViewer/webgl-utils.js

```diff
@@ -110,7 +115,7 @@ var WebGLUtils = function () {
         /** function:(msg) */
         opt_onError ) {
         function handleCreationError( msg ) {
-            if ( msg.indexOf( 'WebGL2' ) !== -1 ) return;
+            if ( !doReportCreationError ) return;
```

Comment:
Yeah I get it, just handling state with global this way it not very convenient especially when there are different code path. but ok